Cabal/changelog

```diff
@@ -54,6 +54,8 @@
 	  into per-component condition trees anyway. Now it's finally been put
 	  out of its misery.
 	* Add `BlockArguments` to `KnownExtension`.
+	* `--hyperlinked-source` now uses Haddock's hyperlinker backend when
```

Comment:
The `cabal haddock` option name is `--hyperlink-source`. We may want to add a `--hyperlinked-source` synonym.

Reply:
@23Skidoo If you don't mind, I would very much like this - I've hit my head on it several times. In fact, could we change the primary option name to `--hyperlinked-source`?

Reply:
Adding a synonym is fine, but I don't think we should change the config setting name, since people have old config files lying around and I don't want to break those without sufficient reason.